db.js

```diff
@@ -0,0 +1,12 @@
+// need to update username + password + dbname
+const {Pool} = require('pg');
+
+const pool = new Pool({
+  user: 'my_user',
+  host: 'localhost',
+  database: 'my_database',
+  password: 'root',
```

Comment:
🚫 **[detect-secrets]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>
* Secret Keyword

Reply:
Sensitive information should be added to .env and then referenced using the env variable